src/app/view/components/_common/dot-icon-button/dot-icon-button.component.spec.ts

```diff
@@ -24,15 +24,15 @@ describe('DotIconButtonComponent', () => {
     it('should render component with dot-icon component and css classes', () => {
         comp.icon = 'test';
         comp.tiny = true;
-        comp.invertColor = true;
+        comp.inverted = true;
```

Comment:
I was thinking about this (because we might use it in other place) I think we don't need this as an `@Input()`

I think we can do it with css like:

```scss
:host {
    &[inverted] {
        button {
            // ...
        }

    }
}
```

I think it worth to try and if it works we should do it in any other component and we save an `Input()` that we only use for styling.

** Same goes for tiny attr **